docker/Dockerfile

```diff
@@ -9,11 +9,10 @@ RUN python3 -m pip install --no-cache-dir cmake
 
 ENV NCCL_LAUNCH_MODE=GROUP
 
-ARG COMMIT_ID=main
+ARG VERSION=main
 
-RUN git clone --depth=1 https://github.com/InternLM/lmdeploy.git &&\
+RUN git clone --depth=1 --branch=${VERSION} https://github.com/InternLM/lmdeploy.git &&\
```

Comment:
We won't create a new branch when a new version is released.

Reply:
It could be a tag. e.g.: `git clone --depth=1 --branch=v0.0.5 https://github.com/InternLM/lmdeploy.git`